test/get_previous_releases.py

```diff
@@ -119,6 +123,13 @@ def download_binary(tag, args) -> int:
     return 0
 
 
+def checksum_exists(version) -> bool:
+    for sum in SHA256_SUMS:
+        if "bitcoin-" + version in SHA256_SUMS[sum]:
+            return True
+    return False
```

Comment:
I think this could be simplified to a one-liner (maybe it could directly be used above then instead of introducing an extra function):
```suggestion
    return "bitcoin-" + version in SHA256_SUMS.values()
```

Reply:
Looking a second time at it, that was too simple-thought; it should be more like
```
    return any("bitcoin-" + version in str for str in SHA256_SUMS.values())
```

Reply:
Yes absolutely! My python is a little bit rusty it's been a while. I didn't know I could do that, it looks cleaner. Thank you!